### DISCUSSION TITLE
Generic activities and customer - projects.

### DISCUSSION THREAD

**Discussion by WhiteTiger-IT** [kimai/kimai#2341] [Questions & Answers]
There are activities that are not related to individual projects or individual customers; for example phone calls, emails, meetings.
However, if I create an entry in the Timesheet I am obliged to select a customer, the projects of THAT customer and the activities of THAT project.
This forces me to have to duplicate all these generic activities on every single customer or, as I am doing now, to report everything to a generic non-existent customer.
But by doing so it is not possible to have a detail of the time spent in phone calls or meetings on a customer or on a single project.
How can this be handled efficiently?

**Comment by kevinpapst** [comment]
Don't select a customer. This field is purely there for filtering the list of projects: it's optional.
The project itself is mandatory yes.
But when you select a project, the list of activities ALWAYS includes all global + all project specific activities.
Global activities wouldn't make much sense, if they couldn't be used...  😁

**Reply by WhiteTiger-IT** [reply]
But I need to identify the activities for each customer and for each project.
For example, yesterday for the same customer I spent almost two hours in email, over an hour in phone calls and two hours and half in two meetings. All this distributed over three projects.
I'd like to make a report to show him how I spend his time both on individual projects and as a total of the various activities.
Customers often do not realize the time wasted unnecessarily, for example in phone calls.
But in Kimai2, if I select the customer, I only see the projects and activities related to him.

**Reply by kevinpapst** [reply]
I don't get the problem. You can filter later on by customer and global activity to show him how much time was spent.
